### PR TITLE
 Add support for high contrast mode and print and no css

### LINKED ISSUE #252: BBC SVG logo doesnt show when printing
**Describe the bug**
When you want to print out an article, onto paper, like its 1976, the icon doesn't show

**To Reproduce**
Steps to reproduce the behaviour:

1. Go to an article with the Brand in
2. Try to print
3. See the svg isn't visible

**Expected behaviour**
The SVG is visible

**Screenshots**
image

**Desktop (please complete the following information):**

- OS: Mac
- Browser: Chrome
- Version: 71.0.3578.98

- [x] Initially labelled with ["bug"](https://github.com/BBC-News/psammead/labels/bug)

### LINKED ISSUE #378: Header - logo SVG with no CSS
If the user doesn’t have CSS (it either hasn't loaded or they have chosen to disable it), it would be awesome if the user can still see the brand logo SVG i.e. a white SVG cannot be seen on a white background. 

The following example code demonstrates how you can display an SVG in black for users without CSS, by specifying an inline fill colour (black) for the SVG which is available if the user doesn’t have CSS (which works on a white background), the colour of the SVG is then changed to white with CSS which can then be seen on the CSS coloured background. An example of how to implement this can be seen here http://2bitstudio.co.uk/code-tests/article/svg-brand.html

**Additional context - High contrast mode on PC**
It's important that the brand logo SVG works with default and custom colour themes (at a minimum with CSS enabled), ensure you test this in development.

### LINKED ISSUE #860: Cannot see BBC logo on high contrast mode in Windows IE
**Describe the bug**
When viewing the page in IE on PC with 'High Contrast White' mode enabled I cannot see the BBC News logo.

It maybe useful to watch this talk for implementation advice https://www.youtube.com/watch?v=CJltSqksCn0 and also read this by Jenny https://confluence.dev.bbc.co.uk/display/~jenny.robertson@bbc.co.uk/Creating+content+for+high+contrast+users

Looking at this quickly, we probably want an inline fill of black (users can then see the SVG on a white background when CSS is disabled), then some CSS to make the fill white when the user has CSS (so this can be seen on the red background and match the design), and also some CSS using a media query for when windows high contrast mode is active to set the fill to 'windowText' so the SVG can be seen in whatever colour theme the user has set (or CSS 'fill: currentColor' without a media query may work - see youtube video at about 30 minutes in).

I would suggest that we decide on an approach for the ZoomText bug before addressing this bug as it may have a knock on effect - And if using an inline fill colour of black while addressing the high contrast bug that we should also make the logo a reasonable size when CSS is disabled - currently it fills the screen.

**To Reproduce**
View an articles page with high-contrast mode on in IE on Windows.

**Expected behavior**
Should be able to see the BBC News logo.

**Desktop (please complete the following information):**

- OS: Windows
- Browser IE
- High-contrast mode

### PR DESCRIPTION
**Pull request by dr3**
Resolves #252
Resolves https://github.com/BBC-News/simorgh/issues/860
Resolves https://github.com/BBC-News/simorgh/issues/378

Ensures the svg is always visible if in high contrast mode, when printing and when css is disabled

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Tests added for new features
- [ ] Test engineer approval

## ORIGINAL
image
## NOW - Windows 7 IE9 Normal
image
## NOW - Windows 7 IE9 High Contrast white
image
## NOW - Windows 7 IE9 High Contrast
image
## NOW - Mac Print
image
## NOW - Mac no css
image

Known problems that will be fixed in other issues -
- Gel grid not working on IE
- SVG is full width when css is disabled

Testing notes - 
The SVG should be visible (white on red background, or black on white background) when:
- In high contrast modes in IE
- When printing the articles
- When CSS is disabled

### CODE REVIEW

File: packages/components/psammead-brand/README.md [hunk 1]
```diff
@@ -20,6 +20,7 @@ The `Brand` component is designed to be used where a BBC logo is required as SVG
 ## Accessibility notes	
 * Visually hidden text is provided (e.g. for screen reader users)
 * `Brand` is keyboard-accessible and provides hover and focus styles
+* The brand SVG has support for users with css disabled or high contrast mode enabled.
```
Comment (ChrisBAshton) [inline packages/components/psammead-brand/README.md:41]: 👍

File: packages/components/psammead-brand/src/index.jsx [hunk 2]
```diff
@@ -77,11 +85,11 @@ const Brand = ({ brandName }) => (
           focusable="false"
           aria-hidden="true"
         >
-          <g fill="#fff" fillRule="evenodd">
+          <SvgGroup fill="#000" fillRule="evenodd">
```
Comment (ChrisBAshton) [inline packages/components/psammead-brand/src/index.jsx:88]: Why the change to `#000` here?

**Review by ChrisBAshton — COMMENTED** [review]
Code looks good, could you add screenshots please?

File: packages/components/psammead-brand/src/index.jsx (hunk 2 above)
Comment (dr3) [reply packages/components/psammead-brand/src/index.jsx:88]: Its the default for when CSS is disabled

**Review by radiocontrolled — DISMISSED** [review]
LGTM. Can you please add a screenshots for when CSS isn't disabled (windows only) to confirm the non-high contrast mode behaviour hasn't changed.

**Review by bcmn — DISMISSED** [review]

**Review by pjlee11 — APPROVED** [review]

**Review by bcmn — APPROVED** [review]

**Comment by greenc05** [comment]
@dr3 does this work for setting custom modes (colours) in High Contrast Mode on PC?

**Comment by dr3** [comment]
@greenc05 It uses the default font colour that the user assigns
image

**Comment by greenc05** [comment]
Fabulous, great work @dr3

**Comment by jamesbrumpton** [comment]
As we have no means of testing this on a local Psammead environment (excluding Storybook, which wouldn't give an accurate representation as it's an iFrame on a page), we're going to merge this then push to Test. We're going to test the logic on Test.
